### PR TITLE
ROX-15965: Add filtered/derived icons to namespace group labels

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

This PR adds an icon to the labels depending on whether the namespace is filtered or derived.
1. A namespace is `filtered` if it was selected using the dropdown. 
2. A namespace is `derived` if it was not filtered and some deployment within a filtered namespace is communicating with some deployment from that namespace

Screenshot 2023-03-16 at 3 47 19 PM

### CODE REVIEW

**Comment by sachaudh** [comment]
@zhenpesky the icons seem to have this circle border around them. I was hoping you could re-upload the SVGs to the jira without those. Let me know if you can, thanks!

**Review by zhenpesky — COMMENTED** [review]
@sachaudh I just uploaded the new SVG format icon for the derived and filtered in the Jira, please check.

File: ui/apps/platform/src/Containers/NetworkGraph/utils/modelUtils.tsx [hunk 1]
```diff
@@ -45,14 +48,16 @@ function getBaseNode(id: string): CustomNodeModel {
...
+    deploymentId: string,
+    isFiltered: boolean
 ): NamespaceNodeModel {
     const namespaceData: NamespaceData = {
         collapsible: true,
         showContextMenu: false,
         type: 'NAMESPACE',
         namespace,
         cluster,
+        labelIconClass: isFiltered ? filteredNamespaceSVG : derivedNamespaceSVG,
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/NetworkGraph/utils/modelUtils.tsx:60]: That's a confusing property name that I guess the topology library dictates.

**Review by vjwilson — APPROVED** [review]

File: ui/apps/platform/src/Containers/NetworkGraph/utils/modelUtils.tsx (hunk 1 above)
Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/utils/modelUtils.tsx:60]: Yes, it is very confusing as it's basically an icon, not a class 😕. Really wish they had some more in-depth docs for these sorts of things

**Pull request merged**